### PR TITLE
Clearify Reolink intial setup

### PR DESCRIPTION
**Pull request by starkillerOG**
## Proposed change

Clearify Reolink intial setup, after PR https://github.com/home-assistant/home-assistant.io/pull/27694 was merged before I could review.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/reolink.markdown [hunk 1]
```diff
@@ -207,15 +207,33 @@ However, these cameras can work with this integration through an NVR in which th
 
 ## Initial Setup
 
-You'll need to configure your new camera/doorbell for your network. If you're using a wired LAN, this is simple enough. Simply plug it in and let it use DHCP to set itself up. Once it's recognized by your network, you can access its configuration with a web browser.
+A brand new Reolink camera first needs to be connected to the network and initialized in which the credentials for the camera are set.
```
Comment (c0ffeeca7) [inline source/_integrations/reolink.markdown]:
```suggestion
A brand new Reolink camera first needs to be connected to the network and initialized. During initialization, the credentials for the camera need to be set.
```

File: source/_integrations/reolink.markdown [hunk 2]
```diff
@@ -207,15 +207,33 @@ However, these cameras can work with this integration through an NVR in which th
...
+There are serveral ways to achieve this.
 
-If you prefer to use Wi-Fi, especially if you want to restrict it to a local IoT network, [here's a quick way to get it connected](https://www.reddit.com/r/reolink/comments/hjn5be/how_to_connect_wifi_without_an_app/). Simply create a QR code with the following XML string:
+### Reolink app/client
+
+The recommended way is to use the [Reolink mobile app, window client or mac client](https://reolink.com/software-and-manual/), just follow the on screen instructions and use the credentials you configure in the Reolink app/client also in Home Assistant.
```
Comment (c0ffeeca7) [inline source/_integrations/reolink.markdown]:
```suggestion
The recommended way is to use the [Reolink mobile app, Windows, or Mac client](https://reolink.com/software-and-manual/). Follow the on-screen instructions.  In Home Assistant, use the credentials you just configured in the Reolink app/client.
```

File: source/_integrations/reolink.markdown [hunk 3]
```diff
@@ -207,15 +207,33 @@ However, these cameras can work with this integration through an NVR in which th
...
+
+### Web browser
+
+When your camera has a LAN port (most WiFi cameras also have a LAN port), first connect the camera to your network using a LAN cable.
```
Comment (c0ffeeca7) [inline source/_integrations/reolink.markdown]:
```suggestion
When your camera has a LAN port (most Wi-Fi cameras also have a LAN port), first connect the camera to your network using a LAN cable.
```

File: source/_integrations/reolink.markdown [hunk 4]
```diff
@@ -207,15 +207,33 @@ However, these cameras can work with this integration through an NVR in which th
...
+Find the IP address of the camera (e.g. in your router) and go to the IP address in a web browser.
```
Comment (c0ffeeca7) [inline source/_integrations/reolink.markdown]:
```suggestion
Find the IP address of the camera (for example by checking in your router) and go to the IP address in a web browser.
```

File: source/_integrations/reolink.markdown [hunk 5]
```diff
@@ -207,15 +207,33 @@ However, these cameras can work with this integration through an NVR in which th
...
+Follow the on screen instructions to first setup the credentials (use the same credentials in Home Assistant).
```
Comment (c0ffeeca7) [inline source/_integrations/reolink.markdown]:
```suggestion
Follow the on-screen instructions to first setup the credentials (use the same credentials in Home Assistant).
```

File: source/_integrations/reolink.markdown [hunk 6]
```diff
@@ -207,15 +207,33 @@ However, these cameras can work with this integration through an NVR in which th
...
+If it is a WiFi camera, go to settings (gear icon) -> Network and fill in your wifi SSID and password, after that you can disconnect the LAN cable and the camera will automatically switch to the WiFi connection.
```
Comment (c0ffeeca7) [inline source/_integrations/reolink.markdown]:
```suggestion
If it is a Wi-Fi camera, go to settings (gear icon) > **Network** and fill in your Wi-Fi SSID and password. After that you can disconnect the LAN cable and the camera will automatically switch to the Wi-Fi connection.
```

File: source/_integrations/reolink.markdown [hunk 7]
```diff
@@ -207,15 +207,33 @@ However, these cameras can work with this integration through an NVR in which th
...
+Now setup the Reolink Home Assistant integration using the credentials you just specified.
```
Comment (c0ffeeca7) [inline source/_integrations/reolink.markdown]:
```suggestion
Now set up the Reolink Home Assistant integration using the credentials you just specified.
```

File: source/_integrations/reolink.markdown [hunk 8]
```diff
@@ -207,15 +207,33 @@ However, these cameras can work with this integration through an NVR in which th
...
+
+### QR code
+
+You can also connect a WiFi camera using a self made QR code, once connected follow the inscructions under "Web browser".
```
Comment (c0ffeeca7) [inline source/_integrations/reolink.markdown]:
```suggestion
You can also connect a Wi-Fi camera using a self-made QR code. Once connected, follow the instructions under **Web browser**.
```

File: source/_integrations/reolink.markdown [hunk 9]
```diff
@@ -207,15 +207,33 @@ However, these cameras can work with this integration through an NVR in which th
...
+Create a QR code using ISO-8859-1 character encoding (not UTF-8) with the following XML string:
 
     <QR><S>ssid</S><P>password</P><C>last4</C></QR>
 
-Use the `ssid` and `password` of your IoT network. The `last4` is the last 4 digits of the QR code used for the Reolink setup. It's printed under the QR code. You can also scan the QR code and grab the last 4 digits.
+Use the `ssid` and `password` of your WiFi network.
```
Comment (c0ffeeca7) [inline source/_integrations/reolink.markdown]:
```suggestion
Use the `ssid` and `password` of your Wi-Fi network.
```

File: source/_integrations/reolink.markdown [hunk 10]
```diff
@@ -207,15 +207,33 @@ However, these cameras can work with this integration through an NVR in which th
...
+The `last4` are the last 4 digits of the QR code which is printed (on the underside) of the camera itself.
+Normally the digits are printed directly under the QR code, alternatively you could scan the QR code and grab the last 4 digits.
```
Comment (c0ffeeca7) [inline source/_integrations/reolink.markdown]:
```suggestion
Normally, the digits are printed directly under the QR code. Alternatively, you could scan the QR code and grab the last 4 digits.
```

File: source/_integrations/reolink.markdown [hunk 11]
```diff
@@ -207,15 +207,33 @@ However, these cameras can work with this integration through an NVR in which th
...
 
-Then power up the camera while pointing it at the QR code. It takes about a minute to initialize, read the QR code, and attach.
+Then power up the camera while pointing it at the QR code. It takes about a minute to initialize, read the QR code, and connect to your WiFi.
```
Comment (c0ffeeca7) [inline source/_integrations/reolink.markdown]:
```suggestion
Then power up the camera while pointing it at the QR code. It takes about a minute to initialize, read the QR code, and connect to your Wi-Fi.
```

**Review by c0ffeeca7 — CHANGES_REQUESTED** [review]
Nice! Thank you for the additional explanation, @starkillerOG!
Some style comments.

**Comment by starkillerOG** [comment]
@c0ffeeca7 thank you for the improvements!

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @starkillerOG! :+1:

**Pull request merged**